feathr_project/feathr/feature_derivations.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import annotations
+from typing import List, Optional
+
+from copy import copy, deepcopy
+from jinja2 import Template
+from typing import Union
+from feathr.dtype import FeatureType, ValueType
+from feathr.feature import Feature
+from feathr.typed_key import DUMMY_KEY, TypedKey
+from feathr.transformation import ExpressionTransformation, RowTransformation
+
+class DerivedFeature:
```

Comment:
Feature and Derived feature should have a base class that have: feature type, feature name, feature description.

Reply:
+1